pkg/apis/core/validation/project.go

```diff
@@ -61,8 +64,22 @@ func ValidateProjectUpdate(newProject, oldProject *core.Project) field.ErrorList
 func ValidateProjectSpec(projectSpec *core.ProjectSpec, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 
+	ownerFound := false
+
 	for i, member := range projectSpec.Members {
-		allErrs = append(allErrs, ValidateSubject(member.Subject, fldPath.Child("members").Index(i))...)
+		idxPath := fldPath.Child("members").Index(i)
+
+		allErrs = append(allErrs, ValidateProjectMember(member, idxPath)...)
+
+		for j, role := range member.Roles {
+			if role == core.ProjectMemberOwner {
+				if ownerFound {
+					allErrs = append(allErrs, field.Forbidden(idxPath.Child("roles").Index(j), role))
```

Comment:
is only one owner allowed? if yes, why?

Reply:
> Is it sufficiently described for you to understand what needs to be done?

yes, thanks

Reply:
Hm yes, mainly because it was like this before. We can remove the `.spec.owner` field in `v1` and then allow to have multiple owners. WDYT?

Reply:
Sounds good